tests/php/test-class-amp-template-customizer.php

```diff
@@ -288,11 +288,6 @@ public function test_init_legacy_preview_controls() {
 		$wp_customize = $this->get_customize_manager();
 		$instance     = AMP_Template_Customizer::init( $wp_customize );
 		$instance->init_legacy_preview();
-		if ( version_compare( strtok( get_bloginfo( 'version' ), '-' ), '5.7', '<' ) ) {
-			$this->assertEquals( 10, has_action( 'amp_post_template_head', 'wp_no_robots' ) );
-		} else {
-			$this->assertFalse( has_action( 'amp_post_template_head', 'wp_no_robots' ) );
```

Comment:
Shouldn't this assertion be kept?

Reply:
We don't have any occurrence where `wp_no_robots` is being hooked to `amp_post_template_head`. I don't think we need to assert that because this assertion will always be true.